src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/assets/SdkDiffExclusions.txt

```diff
@@ -84,3 +84,8 @@ msft,./sdk/x.y.z/FSharp/runtimes/win/lib/netx.y/Microsoft.Win32.SystemEvents.dll
 msft,./sdk/x.y.z/FSharp/runtimes/win/lib/netx.y/System.Drawing.Common.dll
 msft,./sdk/x.y.z/FSharp/runtimes/win/lib/netx.y/System.Security.Cryptography.ProtectedData.dll
 msft,./sdk/x.y.z/FSharp/runtimes/win/lib/netx.y/System.Windows.Extensions.dll
+
+# windows components - https://github.com/dotnet/source-build/issues/3526
+msft,./sdk/x.y.z/runtimes/win/lib/netx.y/Microsoft.Win32.SystemEvents.dll
+msft,./sdk/x.y.z/runtimes/win/lib/netx.y/System.Drawing.Common.dll
+msft,./sdk/x.y.z/runtimes/win/lib/netx.y/System.Windows.Extensions.dll
```

Comment:
Why are these showing up in the Microsoft SDK Linux tarball at all? It seems there should be an issue logged to address this but I don't see one linked from https://github.com/dotnet/source-build/issues/3526

Reply:
This is likely coming from a nuget package that's multi-platform. In some cases `sdk` or `installer` repos would do custom layout processing to remove what's not applicable on Linux. In other cases it did not make sense, therefore we have, for instance, the following `net472` content excluded from SDK diffs: https://github.com/dotnet/installer/blob/0c50e2764ecf3ea0884189cb65c01670d91d427f/src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/assets/SdkDiffExclusions.txt#L17-L32

This issue was tracking for Preview 6. @MichaelSimons are you OK with moving this to 'Blocked' and opening a dependent issue, instead?

Reply:
Created the issue in `sdk` repo and linked to it in the `source-build` issue.

https://github.com/dotnet/sdk/issues/33472

@mthalman I have kept the link in the exclusion list to point to the `source-build` issue as that was a pattern we used in the past.